scss/variables/colors.js

```diff
@@ -24,7 +25,8 @@ chassis.colors = {
 			"base": "#3F51B5",
 			"light": "#5C6BC0",
 			"dark": "#3949AB",
-			"darker": "#303F9F"
+			"darker": "#303F9F",
+			"notification": "#3F51B5"
```

Comment:
I agree with @geekman-rohit, I don't think we should be adding another variable like this indicating the notification color, especially if it's often the same as the base.